### DISCUSSION TITLE
User Role Identification

### DISCUSSION THREAD

**Discussion by pradeepkumarcm96** [Sunbird-coKreat/Community#32] [General Discussions]
@pallakartheekreddy 
How do we identify a user is sourceAdmin or contributorOrgAdmin while logging in into VDN staging portal?

cc -  @kiranharidas187 @githubpiyush, @vijiurs @aks30

**Comment by Nadiya496** [comment]
Hi @pradeepkumarcm96 
1. While login to the cokreat sourcing portal, if the user (**ORG_ADMIN of consumption portal**) can see **Create new Project Button, My Projects, Manage users and Organisation reports.**
2. For the contribution org admin , **All Projects , My Projects , Manage users and My contents** page will be available.
cc: @pallakartheekreddy @surendrasinghs
